benchmarks/benchmarking_conservative.ipynb

```diff
@@ -725,12 +732,60 @@
     "print(f\"xESMF vs. CDO         - RMSE: {rmse:.5f} mm\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compare the runtime and output of different `nan_threshold` values."
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 8,
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/slevang/miniconda3/envs/xarray-regrid/lib/python3.11/site-packages/h5py/__init__.py:36: UserWarning: h5py is running against HDF5 1.14.3 when it was built against 1.14.2, this may cause problems\n",
+      "  _warn((\"h5py is running against HDF5 {0} when it was built against {1}, \"\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Elapsed time skipna=False: 0.284 seconds\n",
+      "Elapsed time skipna=True: 0.913 seconds\n"
+     ]
+    }
+   ],
+   "source": [
+    "ds_sst = xr.tutorial.open_dataset(\"ersstv5\")[[\"sst\"]].rename(lon=\"longitude\", lat=\"latitude\").persist()\n",
+    "\n",
+    "new_grid = Grid(\n",
+    "    north=90,\n",
+    "    east=360,\n",
+    "    south=-90,\n",
+    "    west=0,\n",
+    "    resolution_lat=4,\n",
+    "    resolution_lon=4,\n",
+    ")\n",
+    "target_dataset = create_regridding_dataset(new_grid)\n",
+    "\n",
+    "t0 = time()\n",
+    "data_skipna_false = ds_sst.regrid.conservative(target_dataset, skipna=False).compute()\n",
+    "print(f\"Elapsed time skipna=False: {time() - t0:.3f} seconds\")\n",
+    "\n",
+    "t0 = time()\n",
+    "data_nt0 = ds_sst.regrid.conservative(target_dataset, skipna=True, nan_threshold=0).compute()\n",
+    "print(f\"Elapsed time skipna=True: {time() - t0:.3f} seconds\")\n",
+    "\n",
+    "data_nt1 = ds_sst.regrid.conservative(target_dataset, skipna=True, nan_threshold=1).compute()\n",
+    "\n",
```

Comment:
The nan_threshold=1 computation is not being timed here. Can you add that? (or leave it out if it's the same).